webrtc-stats.html

```diff
@@ -2978,6 +2978,11 @@ <h2>
                   relevant candidate would be <code>"wifi"</code>, even when the next hop is over a
                   cellular connection.
                 </div>
+                <p class="fingerprint">
+                  This reveals information that would otherwise not be available to web pages,
+                  which can increase the fingerprint surface. The user agent SHOULD expose this
+                  field only if the user provides some form of consent.
```

Comment:
I don't think we should have SHOULD statements inside a "fingerprint" note - it's easy to overlook.
Conditional access to stats is something we need to be pretty explicit about if we're going to have any hope of making consistent behaviour out of it. So I'd recommend just going with the fingerprint warning here.